src/panels/lovelace/hui-root.ts

```diff
@@ -228,7 +228,18 @@ class HUIRoot extends LitElement {
                   >
                     ${this.lovelace!.config.views.map(
                       (view) => html`
-                        <paper-tab aria-label="${view.title}">
+                        <paper-tab
+                          aria-label="${view.title}"
+                          class="${classMap({
+                            "hide-tab": Boolean(
+                              !this._editMode &&
+                                view.hide &&
+                                (view.hide === true ||
+                                  (Array.isArray(view.hide) &&
```

Comment:
We should be able to assume that if `hide` is truthy, that it's a boolean or an array of strings, so no need to check if it's an array ?

Reply:
Someone could mess it up and make an object, I suppose?